packages/dnd-timeline/src/types/resize.ts

```diff
@@ -11,6 +12,7 @@ export interface ResizeMoveEvent {
 export type ResizeEndEvent = ResizeMoveEvent;
 
 export interface ResizeStartEvent {
+	activatorEvent: Event;
```

Comment:
Thank you for your great work! :+1:

Why not use `PointerEvent` here instead of just simple `Event`?
I think only `PointerEvent` will be passed to `ResizeXXXEvent`

Reply:
Hey!
Tried to match `dnd-kits`'s event types, where they use a generic `Event` to prevent limiting interaction to pointer.

For example, in `dnd-kit`, it is possible to control dragging using keyboard input.
To allow for future support of keyboard control, or other accessibility features, I preferred using `Event`.

Reply:
I see. Understood! Thank you!